google/datalab/bigquery/_view.py

```diff
@@ -186,7 +186,7 @@ def results(self, use_cache=True, dialect=None, billing_tier=None):
     """
     output_options = QueryOutput.table(use_cache=use_cache)
     return self._materialization.execute(output_options, dialect=dialect,
-                                               billing_tier=billing_tier).results
+                                               billing_tier=billing_tier).result()
```

Comment:
Oh I misread the diff, didn't notice you made it a non-property. Yes the inherited type's method will override the base's.

The only issue now is we don't have access to the GCP operation's result. I think this is fine for now since we don't see any scenarios that need this off the top of my head.

Btw, are there any more `.results` references that need fixing?? I think at least `_query.py` and `_bigquery.py` will have some.